index.html

```diff
@@ -1003,7 +1003,7 @@ <h2 id="loading" style="visibility: hidden;">Loading...</h2>
         const chatModelList = [
             {
                 groupName: 'OpenAI',
-                models: ['gpt-4-turbo-preview', 'gpt-4', 'gpt-3.5-turbo', 'gpt-4-vision-preview', 'gpt-4-32k'],
+                models: [ 'gpt-4o', 'gpt-4-turbo-preview', 'gpt-4', 'gpt-3.5-turbo', 'gpt-4-vision-preview', 'gpt-4-32k'],
```

Comment:
I think we need to add pricing details to estimator, e.g.               { model: /^gpt-4-*[0-9]*$/, priceIn: 30, priceOut: 60 },
Since 4o has different price.

Reply:
If I find time today, I will do it, maybe tomorrow.